src/pages/GetStarted/CustomWorkspaceTab/DevfileSelector.tsx

```diff
@@ -57,41 +63,52 @@ export class DevfileSelectorFormGroup extends React.PureComponent<Props, State>
     this.devfileLocationRef = React.createRef();
   }
 
-  private onDevfileSelected(meta: che.DevfileMetaData): void {
+  private onDevfileClear(): void {
+    if (this.props.onClear) {
+      this.props.onClear();
+    }
+  }
+
+  private async onDevfileSelected(meta: che.DevfileMetaData): Promise<void> {
```

Comment:
`onDevfileSelected` -> `handleDevfileSelect` ?

Reply:
done

Reply:
I don't see any changes, do you prefer to leave it as is?

Reply:
sorry, problem on my site